src/components/admin/CreateEventForm/index.tsx

```diff
@@ -31,6 +31,8 @@ for (let i = 1; i <= 12; i += 1) {
   hours.push(i);
 }
 
+const suborgs = ['General', 'AI', 'Cyber', 'Design', 'Hack', 'Innovate'];
+
```

Comment:
Why not pass this as a prop down from container

Reply:
The other properties are in the same file as well. For now, I'd say it's fine?